joplin/base/signals.py

```diff
@@ -187,15 +181,31 @@ def get_http_request():
         return None
 
 
+# By creating a signal reciever for each snippet model we have, we can avoid
+# needing to filter out large amounts of unwanted calls in our function logic
```

Comment:
I don't understand that, but it sounds like a good thing. Do you mean a "large amounts of unwanted [create_build_aws] calls" or something else?

Reply:
when I had `@receiver(post_save)` in there without a specified sender I was getting all sorts of silly post_save events from django_session etc.

While I'd like to be able to make it more generic (for all snippets post save do x) having "if you're not all sorts of other django post_save stuff" logic in the fn seemed like a bad plan